Interop/ModelCard/src/main/java/UsageDetails.java

```diff
@@ -0,0 +1,355 @@
+        for (int i = 0; i < considerations.size(); i++) {
+            System.out.println("\t" + (i+1) + ") "+ considerations.get(i));
+        }
+        return("Displayed all added considerations.");
+    }
+
+    @Command(
+            usage = "<String> Adds relevant factor to list of factors."
+    )
+    public String addFactor(CommandInterpreter ci, String factor) {
+        factors.add(factor);
+        return("Added factor to list of factors.");
+    }
+
+    @Command(
+            usage = "<int> Remove factor at specified index (1-indexed)."
+    )
+    public String removeFactor(CommandInterpreter ci, int index) {
+        factors.remove(index-1);
+        return("Removed factor at specified index.");
+    }
+
+    @Command(
+            usage = "Displays all added factors."
+    )
+    public String viewFactors(CommandInterpreter ci) {
+        for (int i = 0; i < factors.size(); i++) {
+            System.out.println("\t" + (i+1) + ") "+ factors.get(i));
+        }
+        return("Displayed all added factors.");
+    }
+
+    @Command(
+            usage = "<String> Adds resource to list of resources."
+    )
+    public String addResource(CommandInterpreter ci, String resource) {
+        resources.add(resource);
+        return("Added resource to list of resources.");
+    }
+
+    @Command(
+            usage = "<int> Remove resource at specified index (1-indexed)."
+    )
+    public String removeResource(CommandInterpreter ci, int index) {
+        resources.remove(index-1);
+        return("Removed resource at specified index.");
+    }
+
+    @Command(
+            usage = "Displays all added resources."
+    )
+    public String viewResources(CommandInterpreter ci) {
+        for (int i = 0; i < resources.size(); i++) {
+            System.out.println("\t" + (i+1) + ") "+ resources.get(i));
+        }
+        return("Displayed all added resources.");
+    }
+
+    @Command(
+            usage = "<String> Records primary contact in case of questions or comments."
+    )
+    public String primaryContact(CommandInterpreter ci, String contact) {
+        primaryContact = contact;
+        return("Recorded primary contact as " + primaryContact + ".");
+    }
+
+    @Command(
+            usage = "<String> Records model's citation."
+    )
+    public String modelCitation(CommandInterpreter ci, String citation) {
+        modelCitation = citation;
+        return("Recorded model citation as " + modelCitation + ".");
+    }
+
+    @Command(
+            usage = "<String> Records model's license."
+    )
+    public String modelLicense(CommandInterpreter ci, String license) {
+        modelLicense = license;
+        return("Recorded model license as " + modelLicense + ".");
+    }
+
+    @Command(
+            usage = "<String> Saves UsageDetails to provided destination file and closes CLI."
+    )
+    public String saveUsageDetails(CommandInterpreter ci, String destinationFile) throws IOException {
```

Comment:
The command should accept `File` as a type rather than wrapping a String.

Reply:
Done as suggested.